packages/react-native-branch/android/build.gradle

```diff
@@ -0,0 +1,78 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'maven'
```

Comment:
```suggestion
```

the `maven` plugin is deprecated in gradle 7 (which is the versioned used in react-native 0.67). maybe we can remove the plugin here. or using `maven-publish` plugin as https://github.com/expo/expo/pull/16080